tests/names/test_org_types.py

```diff
@@ -1,51 +1,58 @@
+import pytest
+from rigour.text.dictionary import Replacer, AhoCorReplacer
 from rigour.names.org_types import replace_org_types_display as replace_display
 from rigour.names.org_types import replace_org_types_compare as replace_compare
 from rigour.names.org_types import extract_org_types, remove_org_types
 from rigour.names.org_types import _normalize_compare
+
 # from rigour.names.org_types import normalize_display
 
 
-def test_display_form():
-    assert replace_display("Siemens Aktiengesellschaft") == "Siemens AG"
-    assert replace_display("Siemens AG") == "Siemens AG"
+@pytest.mark.parametrize("clazz", [Replacer, AhoCorReplacer])
+def test_display_form(clazz):
+    assert replace_display("Siemens Aktiengesellschaft", replacer_class=clazz) == "Siemens AG"
```

Comment:
is ignore_case sufficient for replace_display or do we need to support more potential deviation between forms and names?